IdentityCore/src/webview/systemWebview/session/MSIDASWebAuthenticationSessionHandler.h

```diff
@@ -29,15 +29,10 @@
 #import "MSIDWebviewInteracting.h"
 #import "MSIDConstants.h"
 
-#if !MSID_EXCLUDE_WEBKIT && (__IPHONE_OS_VERSION_MAX_ALLOWED >= 120000 || __MAC_OS_X_VERSION_MAX_ALLOWED >= 101500)
+#if !MSID_EXCLUDE_WEBKIT
 
 NS_ASSUME_NONNULL_BEGIN
 
-#if __IPHONE_OS_VERSION_MAX_ALLOWED >= 130000 || __MAC_OS_X_VERSION_MAX_ALLOWED >= 101500
-API_AVAILABLE(ios(12.0), macCatalyst(13.0), macos(10.15))
```

Comment:
[macCatalsyst(13.0) is the same as macOS 10.15](https://stackoverflow.com/questions/63581114/mac-catalyst-version) so now they are both true